docs/team/ashiqurrah.md

```diff
@@ -13,14 +13,46 @@ It is written in Java.
 
 ##Summary of Contributions
 
-* **New Feature**: `to be added soon`
-
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=ashiqurrah&breakdown=true&sort=groupTitle&sortWithin=title&since=2022-09-16&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
 
-* **Project management**: `to be added soon`
-
-* **Enhancements to existing features**: `to be added soon`
-* **Documentation**: `to be added soon`
-* **Community**: `to be added soon`
-* **Tools**: `to be added soon`
-
+* **Features & Enhancements**: 
+  * Implemented `grade` field feature 
+    * What does it do: To monitor their students' grade progress
+      * Updated `AddCommand` and `AddCommandParser` to include `GradeProgress`
+      * Created `GradeProgressCommand`, `GradeProgressCommandParser`, `GradeProgress` and `GradeProgressList` and the respective tests
+  * Implemented `show` command feature
+      * What does it do: To display the sessions of the specified day
+          * Created `ShowCommand` and `ShowCommandParser` and the respective tests
+      * Justification: This feature enables the user to view his sessions in a day
+      * Note: This feature was done with Phylicia Christel
+  * Implemented `mark` command feature
+    * What does it do: To `mark` students' attendance or homework
+      * Updated `HomeworkList` and `AttendanceList` to include `markAtIndex` that allows marking of specified index in respective list.
+      * Created `MarkCommand`, `MarkCommandParser` and `MarkPersonDescriptor` and the respective tests
+    * Justification: This feature enables the user to monitor the students' attendance and homework completion.
+  * Implemented `unmark` command feature
+    * What does it do: To `unmark` students' attendance or homework
+      * Updated `HomeworkList` and `AttendanceList` to include `unmarkAtIndex` that allows marking of specified index in respective list.
+      * Created `UnmarkCommand`, `UnmarkCommandParser` and `MarkPersonDescriptor` and the respective tests
+    * Justification: This feature enables the user to unmark the students' attendance and homework completion and also provides a way for the user to undo his marking of a particulars' attendance or homework.
+  
+* **Contributions to the UG**: 
+  * Added `mark` command
+  * Added `unmark` command
+  * Added `grade` command
+  * Updated the prefixes summary table to include all prefixes
+
+* **Contributions to the DG**:
+    * Added grade progress feature
+    * Added NFRs
+
+* **Contributions to team-based tasks**:
+  * Setting up Github Repo Page
+  * Updated the [AboutUs](https://ay2223s1-cs2103t-w09-4.github.io/tp/AboutUs.html) page 
```

Comment:
why r u stealing my jobbbb 😭💔 i think i updated it as part of the job allocation for setting up and u guys just committed the pics